### ISSUE TITLE
iframe with fragment in src don't render

### ISSUE THREAD

**Issue by olegshilov** [tauri-apps/tauri#1943]
**Describe the bug**
The problem is when iframe src includes a URI fragment, like `#something`, it won't load when you try to load it from disk.

**To Reproduce**
Minimal reproducible example placed here:
https://github.com/olegshilov/tauri-testing/tree/iframe-issue
Steps to reproduce:
- clone
- install dependencies
- run `yarn start` or `yarn build`

**Expected behavior**
Every iframe with fragments in URL must be rendered correctly 

**Screenshots**
![image](https://user-images.githubusercontent.com/930121/120679760-3487e780-c4a2-11eb-8170-190dbd34ee20.png)

**Platform and Versions (please complete the following information):**
```shell
❯ yarn tauri info
 app:spawn Running "/Users/olegshilov/Documents/tauri-testing/.yarn/unplugged/@tauri-apps-cli-npm-1.0.0-beta.2-71cdee115b/node_modules/@tauri-apps/cli/bin/tauri-cli tauri info" +0ms


Operating System - Mac OS, version 11.4.0 X64

Node.js environment
  Node.js - 15.7.0
  @tauri-apps/cli - 1.0.0-beta.1
  @tauri-apps/api - Not installed

Global packages
  npm - 7.5.3
  yarn - 3.0.0-rc.2.git.20210514.hash-4d53b9eb

Rust environment
  rustc - 1.52.1
  cargo - 1.52.0

App directory structure
/.yarn
/app
/src-tauri
/.git

App
  tauri.rs - 1.0.0-beta.1
build-type - bundle
CSP - default-src blob: data: filesystem: ws: http: https: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
distDir - ../app
devPath - ../app

```

**Comment by shirakaba** [comment]
I don't think this is a Tauri-specific issue, but I decided to try it just through a local Apache web server using Safari 14.1.1 on macOS 10.15.7 out of interest.

I made only three changes to your https://github.com/olegshilov/tauri-testing/tree/iframe-issue/app:

* In `index.html`, I changed the start of each path from `/iframe.html` to `./iframe.html` (because I was in a subdirectory of the web server).
* I added the HTML to `iframe.html` to print out the URLs:
    ```html
    <p>location.href: <span id="loc"></span></p>
    <script>document.getElementById("loc").textContent = location.href</script>
    ```
* In `index.html`, I also increased the iframe width to 500 to see the full URL:

<img width="1367" alt="image" src="https://user-images.githubusercontent.com/14055146/120791266-e3243a80-c52b-11eb-9687-6998829289aa.png">

Curiously it works fully over Apache. Can't immediately explain why it's behaving different in the Tauri context.
